.prettierrc

```diff
@@ -0,0 +1,11 @@
+arrowParens: always
+bracketSpacing: true
+jsxBracketSameLine: false
+printWidth: 100
+proseWrap: always
+requirePragma: false
+semi: true
+singleQuote: true
+tabWidth: 2
+trailingComma: all
+useTabs: false
```

Comment:
I recommend going with the default prettier options and not have a `prettierrc` to be consistent across different quran.com projects.

Reply:
It looks like this is just a copy-paste from the `.eslintrc.yml`, which are the rules outlined in #21. Do you think we should change any rules here (probably best for a follow-up in any case)?

Reply:
+1 for getting rid of `.prettierrc` if no one is hung op on those preferences :)